mqtt/mqtt-broker/src/session/state/mod.rs

```diff
@@ -123,7 +123,9 @@ impl SessionState {
 
     pub fn queue_publish(&mut self, publication: proto::Publication) -> Result<(), Error> {
         if let Some(publication) = self.filter(publication) {
-            self.waiting_to_be_sent.enqueue(publication);
+            if let Some(limit) = self.waiting_to_be_sent.enqueue(publication) {
+                warn!("{}. drop publication {:?}", limit, limit.publication())
```

Comment:
How will publication display in the logs? In my temporary hack I am just showing the topic.

Reply:
`proto::Publication` implements `Debug` but not `Display`. that's why it doesn't work with `{}` but works with `{:?}`.
https://doc.rust-lang.org/std/fmt/

Reply:
Good question, it will dump the payload, do we really want it?

Reply:
Any better suggestions?

Reply:
make info showing only a topic and debug with full dump